core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/NFCFragment.kt

```diff
@@ -183,15 +187,16 @@ class NFCFragment : Fragment() {
         super.onDetach()
     }
 
+    @RequiresApi(Build.VERSION_CODES.O)
```

Comment:
Is this necessary? May i know what is this for?

Reply:
I'm using some API of `LocalDate` to get the starting year and tt requires this version check.

![image](https://user-images.githubusercontent.com/20489253/222337727-1726bfc0-846e-4dfd-93bb-0fb1d79ee96d.png)

Reply:
ahhh cool